src/app/src/components/ConfirmRegistration.jsx

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import { arrayOf, bool, func, string } from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Grid from '@material-ui/core/Grid';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+import AppGrid from './AppGrid';
+import AppOverflow from './AppOverflow';
+import ShowOnly from './ShowOnly';
+
+import {
+    confirmAccountRegistration,
+    resetConfirmAccountRegistration,
+} from '../actions/auth';
+
+import { authLoginFormRoute } from '../util/constants';
+
+const confirmRegistrationStyles = Object.freeze({
+    loadingStyles: Object.freeze({}),
+    errorStyles: Object.freeze({
+        color: 'red',
+    }),
+    successStyles: Object.freeze({}),
+});
+
+class ConfirmRegistration extends Component {
+    componentDidMount() {
+        return this.props.confirmRegistration();
+    }
+
+    componentWillUnmount() {
+        return this.props.resetRegistration();
+    }
+
+    render() {
+        const {
+            error,
+            fetching,
+            hasLoggedInDuringSameSession,
+        } = this.props;
+
+        const insetComponent = (() => {
+            if (fetching) {
+                return (
+                    <div style={confirmRegistrationStyles.loadingStyles}>
+                        <CircularProgress />
+                    </div>
+                );
+            }
+
+            if (error && error.length) {
+                return (
+                    <ul style={confirmRegistrationStyles.errorStyles}>
+                        {
+                            error
+                                .map(err => (
+                                    <li key={err}>
+                                        {err}
+                                    </li>
+                                ))
+                        }
+                    </ul>
+                );
+            }
+
+            return (
+                <div style={confirmRegistrationStyles.successStyles}>
+                    <p>
+                        Account was succesfully confirmed!
```

Comment:
Consider adding some links to the success message to guide people to what they could do next. Something like:

```
You can <a>contribute</a> a facility list or <a>search</a> the existing facilities. 
```

Reply:
Sounds good. I added this in 
f7494d4 

![Screen Shot 2019-03-12 at 3 33 02 PM](https://user-images.githubusercontent.com/4165523/54230476-dcd5be00-44dc-11e9-91a7-f439f4127f92.png)